packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmListTests.kt

```diff
@@ -36,6 +36,8 @@ import io.realm.kotlin.query.RealmResults
 import io.realm.kotlin.query.find
 import io.realm.kotlin.test.assertFailsWithMessage
 import io.realm.kotlin.test.platform.PlatformUtils
+import io.realm.kotlin.test.ErrorCatcher
```

Comment:
Pretty much all changes in this file are due to better alignment with the exhaustive testing framework and the logic of the tests has not changed.